samples/components/oauth/redshift.azure.xml

```diff
@@ -50,10 +58,6 @@
       <key>REFRESHTOKEN</key>
       <value>refresh_token</value>
     </entry>
-    <entry>
-      <key>access-token-issue-time</key>
```

Comment:
we have to keep the entry here(at least the key here), even if issued_at is not a valid response from azure. b/c the logic in oauth service is we leverage the map keys to decide the response keys from oauth service: https://gitlab.tableausoftware.com/connectivity/oauth-service-impl/-/blob/master/src/main/java/com/tableau/oauth/configs/PluginOAuthConfig.java?ref_type=heads#L215, and we have fallback logic to set default access-token-issue-time: https://gitlab.tableausoftware.com/connectivity/oauth-service-impl/-/blob/master/src/main/java/com/tableau/oauth/requestors/AuthorizationCodeTokenRequest.java#L74.

Hence, if we remove the entry here, oauth service won't return any access-token-issue-time as part of response into monolith, which means it will force refreshing access token everytime when we hit validateorrefreshaccesstoken method: https://sourcegraph.prod.tableautools.com/development/maestro-2018-1-dev/-/blob/modules/connectors/tabconnectorsbase/main/parser/OAuthValidator.cpp?L59(it will fail validate access token immediately).

We should keep the entry here, we can just set the value to empty or something tho

Reply:
Well that is surprising, the docs say those aren't required. https://tableau.github.io/connector-plugin-sdk/docs/oauth

Reply:
Well...not that surprising.

Reply:
yeah we need to update the doc - it won't break anything tho if you don't have access-token-issue-time and access-token-expires-in in config, just tedious to force refresh access token always

Reply:
I can create a followup story to update https://tableau.github.io/connector-plugin-sdk/docs/oauth

Reply:
nvmd you just created it, thanks!

Reply:
Oh actually I guess this is the right docs, but still not there. https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow#successful-response-2

Reply:
right, there is no "issued_at" returned from azure side, but what i meant is we have to keep the <key> part to make it work, like this mapping would work as well:

Reply:
yeah right, so this entry would work:
<entry>
      <key>access-token-issue-time</key>
      <value>any-random-value</value>

since there is no issued_at returned at all from azure side

Reply:
Also thanks for catching this!